src/ApiManagement/ApiManagement/Commands/NewAzureApiManagement.cs

```diff
@@ -133,6 +133,25 @@ public class NewAzureApiManagement : AzureApiManagementCmdletBase
             " This also enables the ability to authenticate the certificate in the policy on the gateway.")]
         public SwitchParameter EnableClientCertificate { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "A list of availability zones denoting where the api management service is deployed into.")]
+        [ValidateNotNullOrEmpty]
+        public string[] Zone { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Flag only meant to be used for Premium SKU ApiManagement Service and Non Internal VNET deployments. " +
+            "This is useful in case we want to take a gateway region out of rotation." +
+            " This can also be used to standup a new region in Passive mode, test it and then make it Live later.")]
+        public bool? DisableGateway { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Control Plane Apis version constraint for the API Management service.")]
+        public string ApiVersionConstraint { get; set; }
```

Comment:
I believe this string represents the **minimal** API version.
We should name it for example `MinimalApiVersion` or it's too unclear